SvManoylo/HW1/2.py

```diff
@@ -0,0 +1,24 @@
+print('add your value a')
+a = int(input ())
```

Comment:
better without the print method and see for this:
a = int(input ('add your value a'))